packages/block-editor/src/components/list-view/block-contents.js

```diff
@@ -62,27 +55,8 @@ const ListViewBlockContents = forwardRef(
 		const isBlockMoveTarget =
 			blockMovingClientId && selectedBlockInBlockEditor === clientId;
 
-		const {
-			rootClientId: dropTargetRootClientId,
-			clientId: dropTargetClientId,
-			dropPosition,
-		} = blockDropTarget;
-
-		const isDroppingBefore =
-			dropTargetRootClientId === rootClientId &&
-			dropTargetClientId === clientId &&
-			dropPosition === 'top';
-		const isDroppingAfter =
-			dropTargetRootClientId === rootClientId &&
-			dropTargetClientId === clientId &&
-			dropPosition === 'bottom';
-		const isDroppingToInnerBlocks =
-			dropTargetRootClientId === clientId && dropPosition === 'inside';
-
 		const className = classnames( 'block-editor-list-view-block-contents', {
-			'is-dropping-before': isDroppingBefore || isBlockMoveTarget,
-			'is-dropping-after': isDroppingAfter,
-			'is-dropping-to-inner-blocks': isDroppingToInnerBlocks,
+			'is-dropping-before': isBlockMoveTarget,
```

Comment:
TODO - remove this and use the new drop indicator for the 'Move To' option.

Reply:
Hmm, I'm going to make this a follow up too. I think Move To needs some refactoring. It currently changes the selected block to determine where it should be inserted, but it should just use the `insertionPoint` too.